build.gradle

```diff
@@ -39,8 +39,24 @@ subprojects {
 	apply from: "${rootDir}/gradle/xtend-compiler-settings.gradle"
 	apply from: "${rootDir}/gradle/maven-deployment.gradle"
 	apply from: "${rootDir}/gradle/manifest-gen.gradle"
+
+	configurations.all {
+		if (name.endsWith("ToolingMain") || name.endsWith("ToolingTest")) {
```

Comment:
Why not `contains("Tooling")`?

Reply:
should also work.
unfortunately this workaround that works local, does not work on server

Reply:
now it seems to work fine